.travis.yml

```diff
@@ -11,9 +11,9 @@ python:
 matrix:
   include:
   - python: 3.6
-    env: LC_ALL=C LC_CTYPE=C
+    env: LANG=C
```

Comment:
I seem to recall having once started with this flag and later found I had to use LC_ALL and LC_CTYPE. In 4660f43, I made the change. I don't recall why I made that change though. Do you have any reference that indicates that this is the proper mechanism?